app/views/collections/_datacite_search.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_tag('datacite_search', method: :get, remote: true) do %>
+  <%= label_tag(:q, "Datacite Metadata Search: ") %>
+  <%= text_field_tag :q %>
+  <%= hidden_field_tag :fl, "doi, creator, publisher, publicationYear, title" %>
+  <%= hidden_field_tag :rows, "20" %>
+  <%= hidden_field_tag :wt, "json" %>
+  <%= submit_tag("Show up to 20 most relevant results") %>
```

Comment:
This should probably just say 'Search'. The user probably assumes we're showing the most relevant results, and since they can't change the number, I don't think they'll care.